### PR TITLE
built download component

### PR DESCRIPTION
**Pull request by Arielpopcorn**
Hi, Delfi !
I don't have sketch installed, so I can't measure the margin of the designs accurately.
So I made it by eye, so tell me if it's good enough :)

### CODE REVIEW

File: src/pages/Downloads.js [hunk 1]
```diff
@@ -1,22 +1,121 @@
...
+  font-size: 46px;
+  color: #fff;
+`
+
+const Content = styled.div`
+  display: flex;
+  margin-top: 40px;
+`
+
+const P = styled.p`
```
Comment (delfipolito) [inline src/pages/Downloads.js:47]: Maybe this styled component could desappear and we can put this padding inside Description component

File: src/pages/Downloads.js [hunk 2]
```diff
@@ -1,22 +1,121 @@
...
+              <DownloadLink src={FramerSVG} text={"Download for Framer X"} href={"#"}/>
+            </DownloadWrapper>
+          </Content>
+        </PageWrapper>
+      </Container>
     </Section>
   </Page>
 )
 
+const DownloadLink = (props) =>
```
Comment (delfipolito) [inline src/pages/Downloads.js]: We can separate the two components to be on different files. I would put DownloadLink inside src/components/Downloads

File: src/pages/Downloads.js [hunk 3]
```diff
@@ -1,22 +1,121 @@
...
+  display: flex;
+  text-decoration: none;
+
+`
+
+const LinkText = styled.p`
+  font-size: 17px;
+  color: #6DCBFE;
+  padding-left: 20px;
+  padding-right: 20px;
```
Comment (delfipolito) [inline src/pages/Downloads.js]: Maybe a little less padding-right on LinkText? I would give it 7px

**Review by delfipolito — COMMENTED** [review]
Hey Ariel, it’s looking very good! 🤩

I added some comments, mostly about coding style.

One non trivial change we should do before merging would be to make the component responsive. We should move downloads links to bottom when its on mobile 😊

**Comment by Arielpopcorn** [comment]
Thank you for the feed back😍

I didn’t see the mobile design, is it in the sketch files? If so, Can you send me a screenshot like before, and I’ll get that done:)

File: src/pages/Downloads.js (hunk 1 above)
Comment (Arielpopcorn) [reply src/pages/Downloads.js:47]: I made that cos the two paragraphs need a gap between each other. 
Is it ok if I leave that in? :)

Or maybe did you mean have a p selector inside Description? Like this?

```js
const Description = styled.p`
  font-size: 17px;
  width: 466px;
  color: #fff;

  p {
    padding-bottom: 30px;
  }
`
```

File: src/pages/Downloads.js (hunk 2 above)
Comment (Arielpopcorn) [reply src/pages/Downloads.js]: ok! I moved them to src/components/DownloadLink, looks cleaner :)

File: src/pages/Downloads.js (hunk 3 above)
Comment (Arielpopcorn) [reply src/pages/Downloads.js]: I changed it:)
And one question is, if the screen width below 680px the links will look terrible, is that ok?

**Pull request merged**